ht2/dz.css

```diff
@@ -0,0 +1,145 @@
+/*переподключить шрифт!!!!!!!!!!!!!*/
+body {
+	font-family: OpenSansRegular, serif, sans-serif;
+	font-size: 13px;
+	background-color: #f9f9f9;	
+}
+/*.....................Сама страница........................*/
+* {
+	padding: 0;
+	margin: 0;
+}
+a {
+	text-decoration: none;
+}
+li {
+	list-style-type: none;
+}
+.wrapper {
+	margin: auto;
+	padding: 50px 48px 60px 46px;
+	width: 1060px;	
+	box-shadow: 0 5px 5px rgba(0, 0, 0, 0.3);
+}
+/*..........................HEADER...........................*/
+/*TOP MENU*/
+.top-menu {
+	display: flex;
+	background-color: #ffffff;
+	background-image: linear-gradient(to top, #444343 0%, #5e5e5e 100%);
+	margin-top: 40px;
+}
+.top-menu_item {
+	display: block;
+	padding: 14px 25px 11px;
```

Comment:
The best way to align text vertically when you have one line - is to have line-height===height.
and no vertical paddings. Try it out.